controllers/defaults.go

```diff
@@ -42,7 +42,8 @@ var (
 
 	TopolvmCSIDriverName = "topolvm.cybozu.com"
 
-	VGManagerImage = GetEnvOrDefault("VGMANAGER_IMAGE")
+	VGManagerImage          = GetEnvOrDefault("VGMANAGER_IMAGE")
+	VGManagerServiceAccount = "vg-manager"
```

Comment:
Please move these changes to the "add vgmanager" commit

Reply:
done.